2025-06-01 14:39:45,349:DEBUG:certbot._internal.main:certbot version: 2.10.0
2025-06-01 14:39:45,349:DEBUG:certbot._internal.main:Location of certbot entry point: /usr/bin/certbot
2025-06-01 14:39:45,349:DEBUG:certbot._internal.main:Arguments: ['-v', '--webroot', '--agree-tos', '--renew-by-default', '--preferred-challenges', 'http-01', '--email', 'vaxelaire.yohem@gmail.com', '--webroot-path', '/var/www/certbot', '-d', 'hackthebox.vaxelaire.fr', '-n']
2025-06-01 14:39:45,349:DEBUG:certbot._internal.main:Discovered plugins: PluginsRegistry(PluginEntryPoint#manual,PluginEntryPoint#null,PluginEntryPoint#standalone,PluginEntryPoint#webroot)
2025-06-01 14:39:45,410:DEBUG:certbot._internal.log:Root logging level set at 20
2025-06-01 14:39:45,411:DEBUG:certbot._internal.plugins.selection:Requested authenticator webroot and installer None
2025-06-01 14:39:45,411:DEBUG:certbot._internal.plugins.selection:Single candidate plugin: * webroot
Description: Saves the necessary validation files to a .well-known/acme-challenge/ directory within the nominated webroot path. A seperate HTTP server must be running and serving files from the webroot path. HTTP challenge only (wildcards not supported).
Interfaces: Authenticator, Plugin
Entry point: EntryPoint(name='webroot', value='certbot._internal.plugins.webroot:Authenticator', group='certbot.plugins')
Initialized: <certbot._internal.plugins.webroot.Authenticator object at 0x7f9070621190>
Prep: True
2025-06-01 14:39:45,412:DEBUG:certbot._internal.plugins.selection:Selected authenticator <certbot._internal.plugins.webroot.Authenticator object at 0x7f9070621190> and installer None
2025-06-01 14:39:45,412:INFO:certbot._internal.plugins.selection:Plugins selected: Authenticator webroot, Installer None
2025-06-01 14:39:45,474:DEBUG:certbot._internal.main:Picked account: <Account(RegistrationResource(body=Registration(key=None, contact=(), agreement=None, status=None, terms_of_service_agreed=None, only_return_existing=None, external_account_binding=None), uri='https://acme-v02.api.letsencrypt.org/acme/acct/2243553505', new_authzr_uri=None, terms_of_service=None), b674a225a17192af95a621d2ec1bc036, Meta(creation_dt=datetime.datetime(2025, 2, 22, 10, 55, 38, tzinfo=<UTC>), creation_host='faa0276bdd8e', register_to_eff=None))>
2025-06-01 14:39:45,475:DEBUG:acme.client:Sending GET request to https://acme-v02.api.letsencrypt.org/directory.
2025-06-01 14:39:45,477:DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): acme-v02.api.letsencrypt.org:443
2025-06-01 14:39:45,841:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "GET /directory HTTP/1.1" 200 1012
2025-06-01 14:39:45,842:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 01 Jun 2025 14:39:45 GMT
Content-Type: application/json
Content-Length: 1012
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "SOs8SrThJFc": "https://community.letsencrypt.org/t/adding-random-entries-to-the-directory/33417",
  "keyChange": "https://acme-v02.api.letsencrypt.org/acme/key-change",
  "meta": {
    "caaIdentities": [
      "letsencrypt.org"
    ],
    "profiles": {
      "classic": "https://letsencrypt.org/docs/profiles#classic",
      "shortlived": "https://letsencrypt.org/docs/profiles#shortlived (not yet generally available)",
      "tlsserver": "https://letsencrypt.org/docs/profiles#tlsserver"
    },
    "termsOfService": "https://letsencrypt.org/documents/LE-SA-v1.5-February-24-2025.pdf",
    "website": "https://letsencrypt.org"
  },
  "newAccount": "https://acme-v02.api.letsencrypt.org/acme/new-acct",
  "newNonce": "https://acme-v02.api.letsencrypt.org/acme/new-nonce",
  "newOrder": "https://acme-v02.api.letsencrypt.org/acme/new-order",
  "renewalInfo": "https://acme-v02.api.letsencrypt.org/draft-ietf-acme-ari-03/renewalInfo",
  "revokeCert": "https://acme-v02.api.letsencrypt.org/acme/revoke-cert"
}
2025-06-01 14:39:45,864:DEBUG:certbot._internal.renewal:Auto-renewal forced with --force-renewal...
2025-06-01 14:39:45,864:DEBUG:certbot._internal.display.obj:Notifying user: Renewing an existing certificate for hackthebox.vaxelaire.fr
2025-06-01 14:39:45,866:DEBUG:acme.client:Requesting fresh nonce
2025-06-01 14:39:45,866:DEBUG:acme.client:Sending HEAD request to https://acme-v02.api.letsencrypt.org/acme/new-nonce.
2025-06-01 14:39:45,986:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "HEAD /acme/new-nonce HTTP/1.1" 200 0
2025-06-01 14:39:45,986:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 01 Jun 2025 14:39:45 GMT
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: GRQqzLBfQR0rM8xIDzcrhWtZgmSKRDAjoJ63SYFzd2NgMS29frA
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800


2025-06-01 14:39:45,987:DEBUG:acme.client:Storing nonce: GRQqzLBfQR0rM8xIDzcrhWtZgmSKRDAjoJ63SYFzd2NgMS29frA
2025-06-01 14:39:45,987:DEBUG:acme.client:JWS payload:
b'{\n  "identifiers": [\n    {\n      "type": "dns",\n      "value": "hackthebox.vaxelaire.fr"\n    }\n  ]\n}'
2025-06-01 14:39:45,989:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/new-order:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMjI0MzU1MzUwNSIsICJub25jZSI6ICJHUlFxekxCZlFSMHJNOHhJRHpjcmhXdFpnbVNLUkRBam9KNjNTWUZ6ZDJOZ01TMjlmckEiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL25ldy1vcmRlciJ9",
  "signature": "AutBo12FBm43dxEss6IojNaHagY1uaSkQYs6-XmlfqjYh-3giO79IIkuw5Pogp7l-U13Drx1nT-umTwBXy6tXBysVpjUrpZaV_MDVumdQ1Tpdq5y4mNxGC_sL9UCttrpoCBHYIA0N2B0SwbQ856NsaWgIZ6NXF1tnx_iuUN5U0QdZDzXFO7GSuVkIoQa5NsAtkOFi3WHlbLT7NHdcy9So1lvyJ99YIUDVwQfmTYSTsrVFnpJqlya9HJEawxH5CmdcHgTEkQOE2BA5M0lh_9hCkoDHVs-lKNWucavr8xM5qMSDnKcUdsn1XnrDN2QlZ1bTCdGzjXKk6rk3Tr_ap_XmQ",
  "payload": "ewogICJpZGVudGlmaWVycyI6IFsKICAgIHsKICAgICAgInR5cGUiOiAiZG5zIiwKICAgICAgInZhbHVlIjogImhhY2t0aGVib3gudmF4ZWxhaXJlLmZyIgogICAgfQogIF0KfQ"
}
2025-06-01 14:39:46,229:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/new-order HTTP/1.1" 201 357
2025-06-01 14:39:46,230:DEBUG:acme.client:Received response:
HTTP 201
Server: nginx
Date: Sun, 01 Jun 2025 14:39:46 GMT
Content-Type: application/json
Content-Length: 357
Connection: keep-alive
Boulder-Requester: 2243553505
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Location: https://acme-v02.api.letsencrypt.org/acme/order/2243553505/390105307517
Replay-Nonce: _G_8-Az_jFyq1QOAo1nyjogq4G2PEZVnTrJFRbtS3L-GQghD1gk
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "status": "pending",
  "expires": "2025-06-08T14:39:46Z",
  "identifiers": [
    {
      "type": "dns",
      "value": "hackthebox.vaxelaire.fr"
    }
  ],
  "authorizations": [
    "https://acme-v02.api.letsencrypt.org/acme/authz/2243553505/529253611427"
  ],
  "finalize": "https://acme-v02.api.letsencrypt.org/acme/finalize/2243553505/390105307517"
}
2025-06-01 14:39:46,230:DEBUG:acme.client:Storing nonce: _G_8-Az_jFyq1QOAo1nyjogq4G2PEZVnTrJFRbtS3L-GQghD1gk
2025-06-01 14:39:46,230:DEBUG:acme.client:JWS payload:
b''
2025-06-01 14:39:46,232:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz/2243553505/529253611427:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMjI0MzU1MzUwNSIsICJub25jZSI6ICJfR184LUF6X2pGeXExUU9BbzFueWpvZ3E0RzJQRVpWblRySkZSYnRTM0wtR1FnaEQxZ2siLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LzIyNDM1NTM1MDUvNTI5MjUzNjExNDI3In0",
  "signature": "T1BVHchZYqAq4jienNeno8qGdskIcHu62C2tTdltHW63xs97mLQdJQP6w4Q6qwXRcp66dE60vhMq0InhH3Rg53r_5J6e4onQ5faMWYxVWcViXZc1jjRbbMV7v2Qy6d0_IDsB0oxbef1t-WWvi2FToryVvzbcj6ZOMuj5eiCYMcLUHQhq5jwYOXLywYJfKEXC6Jk3SV6Vx7OZ0xe2R7y3YVPB4891oDy6nJgqwwvK3gHYfqUjDX1EHlxo51ANQfq8oEQcQAoa3rXHJoZN8dgJsedknZn7mIURTaWpmABhR__Ga3VVj9vzenvl3VWkPQ-dHDEgz0rItwb2Ow6S7IGx7w",
  "payload": ""
}
2025-06-01 14:39:46,354:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz/2243553505/529253611427 HTTP/1.1" 200 831
2025-06-01 14:39:46,354:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 01 Jun 2025 14:39:46 GMT
Content-Type: application/json
Content-Length: 831
Connection: keep-alive
Boulder-Requester: 2243553505
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: _G_8-Az_js8LY1UOkh7HMkBEuJ5p3BLGCt4JkYhkJCAqSGGwZ2A
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "hackthebox.vaxelaire.fr"
  },
  "status": "pending",
  "expires": "2025-06-08T14:39:46Z",
  "challenges": [
    {
      "type": "dns-01",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2243553505/529253611427/jx-svw",
      "status": "pending",
      "token": "W0EfuIfYTnd4wNTuYl0jlGcfs569FLEnC5qKjJ0ZcdE"
    },
    {
      "type": "tls-alpn-01",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2243553505/529253611427/oZXrmA",
      "status": "pending",
      "token": "W0EfuIfYTnd4wNTuYl0jlGcfs569FLEnC5qKjJ0ZcdE"
    },
    {
      "type": "http-01",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2243553505/529253611427/-oK4cg",
      "status": "pending",
      "token": "W0EfuIfYTnd4wNTuYl0jlGcfs569FLEnC5qKjJ0ZcdE"
    }
  ]
}
2025-06-01 14:39:46,355:DEBUG:acme.client:Storing nonce: _G_8-Az_js8LY1UOkh7HMkBEuJ5p3BLGCt4JkYhkJCAqSGGwZ2A
2025-06-01 14:39:46,355:INFO:certbot._internal.auth_handler:Performing the following challenges:
2025-06-01 14:39:46,355:INFO:certbot._internal.auth_handler:http-01 challenge for hackthebox.vaxelaire.fr
2025-06-01 14:39:46,355:INFO:certbot._internal.plugins.webroot:Using the webroot path /var/www/certbot for all unmatched domains.
2025-06-01 14:39:46,356:DEBUG:certbot._internal.plugins.webroot:Creating root challenges validation dir at /var/www/certbot/.well-known/acme-challenge
2025-06-01 14:39:46,356:DEBUG:certbot._internal.plugins.webroot:Attempting to save validation to /var/www/certbot/.well-known/acme-challenge/W0EfuIfYTnd4wNTuYl0jlGcfs569FLEnC5qKjJ0ZcdE
2025-06-01 14:39:46,357:DEBUG:acme.client:JWS payload:
b'{}'
2025-06-01 14:39:46,358:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/chall/2243553505/529253611427/-oK4cg:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMjI0MzU1MzUwNSIsICJub25jZSI6ICJfR184LUF6X2pzOExZMVVPa2g3SE1rQkV1SjVwM0JMR0N0NEprWWhrSkNBcVNHR3daMkEiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2NoYWxsLzIyNDM1NTM1MDUvNTI5MjUzNjExNDI3Ly1vSzRjZyJ9",
  "signature": "L3hu6d7gvxeu67-oD-_Tm6tX3ADIcZSMJ8iJTW8LlIaN8yjhPVesNIxXpqrbgtQW5O6dl_s_o2lCznkYUoQMwTQCrX-SCfQj5yTrUb9WZGdPdEQtOWiDJ4iMPofvSmDudCocNH94VCVWeeU-Eewxtbj-98hHe6Eas_VMv0WHssXSoQNw0G8OqXFbdMrpurqhDxPBUSglSSqnJpZrGvL75PuG82klQiB5pBbljQo0lTGjmWAhdMafXcYkGrO1fXalX0YAh5onnhHdzKkfdtzb9v2yWRqBkPvO6JhXV8_dDKWl7tlO1VgOrufXpGnL1PH1bB5-h45xl4Cd_u6G6NFKQw",
  "payload": "e30"
}
2025-06-01 14:39:46,509:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/chall/2243553505/529253611427/-oK4cg HTTP/1.1" 200 195
2025-06-01 14:39:46,510:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 01 Jun 2025 14:39:46 GMT
Content-Type: application/json
Content-Length: 195
Connection: keep-alive
Boulder-Requester: 2243553505
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index", <https://acme-v02.api.letsencrypt.org/acme/authz/2243553505/529253611427>;rel="up"
Location: https://acme-v02.api.letsencrypt.org/acme/chall/2243553505/529253611427/-oK4cg
Replay-Nonce: GRQqzLBfgdADkXVSulb2SwAbrZZRSC6Eon0CGR8Abm67UOgGTJE
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "type": "http-01",
  "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2243553505/529253611427/-oK4cg",
  "status": "pending",
  "token": "W0EfuIfYTnd4wNTuYl0jlGcfs569FLEnC5qKjJ0ZcdE"
}
2025-06-01 14:39:46,510:DEBUG:acme.client:Storing nonce: GRQqzLBfgdADkXVSulb2SwAbrZZRSC6Eon0CGR8Abm67UOgGTJE
2025-06-01 14:39:46,511:INFO:certbot._internal.auth_handler:Waiting for verification...
2025-06-01 14:39:47,511:DEBUG:acme.client:JWS payload:
b''
2025-06-01 14:39:47,512:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz/2243553505/529253611427:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMjI0MzU1MzUwNSIsICJub25jZSI6ICJHUlFxekxCZmdkQURrWFZTdWxiMlN3QWJyWlpSU0M2RW9uMENHUjhBYm02N1VPZ0dUSkUiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LzIyNDM1NTM1MDUvNTI5MjUzNjExNDI3In0",
  "signature": "hUmUeDVwglmHfV08Myn80cMamM1DcPe2a2IKpWmyRpGacgHeqvygaDK_UbBsGsUwgOs7df6fA-fVv2icHFCjlG_6ltNki0ezFEQ_lfBX8KoOeb0-4GBoTB8WJtV6szdjDzDnTPqE7bV5qwRTAqaVxTqM55m7zpabAoSMBxi3IyKlDzeDfrayX1V7TqcrFTnvNWHi1SrS18mZ1wLgujA3JKJwh1ZmYxPGA63NFvbsaueXyiKyvu6lD0hEA6jCVrroc6qaZ81DFm1Re9AExwHDdW9RAiRYQo8T4nuBv307LjZtIXzj-bQgWRdPDmcowVpl2utU6ZRLLSBt4aogAalisA",
  "payload": ""
}
2025-06-01 14:39:47,635:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz/2243553505/529253611427 HTTP/1.1" 200 1489
2025-06-01 14:39:47,636:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 01 Jun 2025 14:39:47 GMT
Content-Type: application/json
Content-Length: 1489
Connection: keep-alive
Boulder-Requester: 2243553505
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: _G_8-Az_6BQNKLt6EWGvpzl2Mj2J6LcyT1_z8SHyb4yJXUwvZyo
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "hackthebox.vaxelaire.fr"
  },
  "status": "invalid",
  "expires": "2025-06-08T14:39:46Z",
  "challenges": [
    {
      "type": "http-01",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2243553505/529253611427/-oK4cg",
      "status": "invalid",
      "validated": "2025-06-01T14:39:46Z",
      "error": {
        "type": "urn:ietf:params:acme:error:connection",
        "detail": "91.108.113.88: Fetching http://hackthebox.vaxelaire.fr/.well-known/acme-challenge/W0EfuIfYTnd4wNTuYl0jlGcfs569FLEnC5qKjJ0ZcdE: Error getting validation data",
        "status": 400
      },
      "token": "W0EfuIfYTnd4wNTuYl0jlGcfs569FLEnC5qKjJ0ZcdE",
      "validationRecord": [
        {
          "url": "http://hackthebox.vaxelaire.fr/.well-known/acme-challenge/W0EfuIfYTnd4wNTuYl0jlGcfs569FLEnC5qKjJ0ZcdE",
          "hostname": "hackthebox.vaxelaire.fr",
          "port": "80",
          "addressesResolved": [
            "91.108.113.88",
            "2a02:4780:28:d15a::1"
          ],
          "addressUsed": "2a02:4780:28:d15a::1"
        },
        {
          "url": "http://hackthebox.vaxelaire.fr/.well-known/acme-challenge/W0EfuIfYTnd4wNTuYl0jlGcfs569FLEnC5qKjJ0ZcdE",
          "hostname": "hackthebox.vaxelaire.fr",
          "port": "80",
          "addressesResolved": [
            "91.108.113.88",
            "2a02:4780:28:d15a::1"
          ],
          "addressUsed": "91.108.113.88"
        }
      ]
    }
  ]
}
2025-06-01 14:39:47,636:DEBUG:acme.client:Storing nonce: _G_8-Az_6BQNKLt6EWGvpzl2Mj2J6LcyT1_z8SHyb4yJXUwvZyo
2025-06-01 14:39:47,637:INFO:certbot._internal.auth_handler:Challenge failed for domain hackthebox.vaxelaire.fr
2025-06-01 14:39:47,637:INFO:certbot._internal.auth_handler:http-01 challenge for hackthebox.vaxelaire.fr
2025-06-01 14:39:47,637:DEBUG:certbot._internal.display.obj:Notifying user: 
Certbot failed to authenticate some domains (authenticator: webroot). The Certificate Authority reported these problems:
  Domain: hackthebox.vaxelaire.fr
  Type:   connection
  Detail: 91.108.113.88: Fetching http://hackthebox.vaxelaire.fr/.well-known/acme-challenge/W0EfuIfYTnd4wNTuYl0jlGcfs569FLEnC5qKjJ0ZcdE: Error getting validation data

Hint: The Certificate Authority failed to download the temporary challenge files created by Certbot. Ensure that the listed domains serve their content from the provided --webroot-path/-w and that files created there can be downloaded from the internet.

2025-06-01 14:39:47,638:DEBUG:certbot._internal.error_handler:Encountered exception:
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/usr/lib/python3.12/site-packages/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.

2025-06-01 14:39:47,638:DEBUG:certbot._internal.error_handler:Calling registered functions
2025-06-01 14:39:47,638:INFO:certbot._internal.auth_handler:Cleaning up challenges
2025-06-01 14:39:47,638:DEBUG:certbot._internal.plugins.webroot:Removing /var/www/certbot/.well-known/acme-challenge/W0EfuIfYTnd4wNTuYl0jlGcfs569FLEnC5qKjJ0ZcdE
2025-06-01 14:39:47,639:DEBUG:certbot._internal.plugins.webroot:All challenges cleaned up
2025-06-01 14:39:47,639:DEBUG:certbot._internal.log:Exiting abnormally:
Traceback (most recent call last):
  File "/usr/bin/certbot", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/usr/lib/python3.12/site-packages/certbot/main.py", line 19, in main
    return internal_main.main(cli_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/certbot/_internal/main.py", line 1894, in main
    return config.func(config, plugins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/certbot/_internal/main.py", line 1600, in certonly
    lineage = _get_and_save_cert(le_client, config, domains, certname, lineage)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/certbot/_internal/main.py", line 131, in _get_and_save_cert
    renewal.renew_cert(config, domains, le_client, lineage)
  File "/usr/lib/python3.12/site-packages/certbot/_internal/renewal.py", line 399, in renew_cert
    new_cert, new_chain, new_key, _ = le_client.obtain_certificate(domains, new_key)
                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/certbot/_internal/client.py", line 428, in obtain_certificate
    orderr = self._get_order_and_authorizations(csr.data, self.config.allow_subset_of_names)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/certbot/_internal/client.py", line 496, in _get_order_and_authorizations
    authzr = self.auth_handler.handle_authorizations(orderr, self.config, best_effort)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/usr/lib/python3.12/site-packages/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.
2025-06-01 14:39:47,642:ERROR:certbot._internal.log:Some challenges have failed.
